package.json

```diff
@@ -13,37 +13,36 @@
   "author": "Algolia <support@algolia.com>",
   "license": "MIT",
   "repository": "algolia/instantsearch.js",
-  "main": "dist-es5-module/index.js",
-  "module": "./es/index.js",
-  "jsdelivr": "dist/instantsearch.min.js",
-  "sideEffects": false,
```

Comment:
Why did we remove this?

Reply:
`sideEffects` is only read by Webpack, right? (not Rollup)

Reply:
Yes but it's for consumers of the lib not us.

Reply:
Ah I see!